File: lab2Main.cs
using System;

public class Lines
{
    private string line;

    // Конструктор за замовчуванням
    public Lines()
    {
        line = "";
    }

    // Конструктор з параметром
    public Lines(string str)
    {
        line = str;
    }

    // Метод для обчислення довжини
    public int Length()
    {
        return line.Length;
    }

    // Метод для видалення символу '5'
    public void RemoveFives()
    {
        line = line.Replace("5", "");
    }

    // Метод для отримання значення рядка
    public string GetValue()
    {
        return line;
    }
}

class Program
{
    static void Main()
    {
        // Створення об'єкта класу Lines
        Lines lineObj = new Lines("1234556789");
        Console.WriteLine("Рядок: " + lineObj.GetValue());

        // Обчислення та виведення довжини рядка
        Console.WriteLine("Довжина рядка: " + lineObj.Length());

        // Видалення символу '5'
        lineObj.RemoveFives();
        Console.WriteLine("Рядок після видалення '5': " + lineObj.GetValue());

        // Обчислення та виведення нової довжини рядка
        Console.WriteLine("Нова довжина рядка: " + lineObj.Length());
    }
}

File: lab2Main.cpp
#include "main.h"

// Конструктор за замовчуванням
Lines::Lines() : line("") {}

// Конструктор з параметром
Lines::Lines(const std::string& str) : line(str) {}

// Метод для обчислення довжини
int Lines::length() const {
    return line.length();
}

// Метод для видалення символу '5'
void Lines::removeFives() {
    line.erase(std::remove(line.begin(), line.end(), '5'), line.end());
}

// Метод для отримання значення рядка
const std::string& Lines::getValue() const {
    return line;
}

int main() {
    // Створення об'єкта класу Lines
    Lines lineObj("1234556789");
    std::cout << "Рядок: " << lineObj.getValue() << std::endl;

    // Обчислення та виведення довжини рядка
    std::cout << "Довжина рядка: " << lineObj.length() << std::endl;

    // Видалення символу '5'
    lineObj.removeFives();
    std::cout << "Рядок після видалення '5': " << lineObj.getValue() << std::endl;

    // Обчислення та виведення нової довжини рядка
    std::cout << "Нова довжина рядка: " << lineObj.length() << std::endl;

    return 0;
}